nusight2/src/client/components/robot_selector/stories/robot_selector.stories.tsx

```diff
@@ -49,24 +49,24 @@ function getRobots(): RobotModel[] {
       name: 'Virtual Robot 1',
       connected: true,
       enabled: true,
-      address: '',
-      port: 0,
+      address: '10.222.10.14',
```

Comment:
I don't understand this change - could you please give context? It seems to me that these are arbitrary addresses that are strange to commit, but I have no idea what's happening here to start with. Is this "story" an example?

Reply:
[Storybook](https://storybook.js.org/docs/react/get-started/introduction) is a UI development tool it allows you to test small components of NUsight and see what they look like without having to implement everything into NUsight in one go. These are just some test cases, some virtual robots, that let you see what it might look like with active connections. As such I just picked some random ip addresses and ports. You can check it out by running `./b yarn storybook`

Reply:
Thanks for the explanation. That makes sense